test/NuGet.Core.Tests/NuGet.Protocol.Tests/PackageSearchResourceV3Tests.cs

```diff
@@ -137,6 +138,7 @@ public async Task PackageSearchResourceV3_GetMetadataAsync_VersionsDownloadCount
             Assert.Equal(28390569, package.DownloadCount);
             Assert.Equal(14, versions.Count());
             Assert.Equal(64099, versions.First().DownloadCount);
+            Assert.Equal(largerThanIntMax, versions.Skip(1).First().DownloadCount);
```

Comment:
The test seems a little unintuitive. You have to know that `EntityFrameworkSearch` has a 2nd version with download count that was too large for an `int32`. Can you add a comment (I prefer block comments above the method) that somehow paraphrases this is a dependency and one of the assertions in this test?

I suggested a comment rather than an Theory or a separate test, as I expect the perf cost would be higher for this one.

Reply:
I just added inline comment here, not above the method, because I have seen before with other cases testing edge cases in middle of other asserts.